shared/chat/conversation/info-panel/participants.js

```diff
@@ -41,7 +41,6 @@ const Participants = ({participants, onShowProfile}: Props) => (
                 {fullname}
               </Text>
             </Box>
-            {index < arr.length - 1 ? <Divider style={{marginLeft: 48}} /> : null}
```

Comment:
ah, did you intend to remove the dividers? if so, please remove the now-unused `index` and `arr` arguments to map above